From 7e01df44401fb6a0aeeecabdb0026290b36f0a03 Mon Sep 17 00:00:00 2001
From: Kevin Wolf <kwolf@redhat.com>
Date: Thu, 20 Nov 2014 13:47:22 +0100
Subject: [PATCH 3/5] Revert "rbd: Only look for qemu-specific copy of
 librbd.so.1"

Message-id: <1416491244-20209-3-git-send-email-kwolf@redhat.com>
Patchwork-id: 62498
O-Subject: [RHEL-7.1 qemu-kvm PATCH v2 2/4] Revert "rbd: Only look for qemu-specific copy of librbd.so.1"
Bugzilla: 1140742
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>
RH-Acked-by: Max Reitz <mreitz@redhat.com>

This reverts commit ece4ff225717364edc3136599113709dacdc5731.

Signed-off-by: Kevin Wolf <kwolf@redhat.com>
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 block/rbd.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/block/rbd.c b/block/rbd.c
index b870401..89319dc 100644
--- a/block/rbd.c
+++ b/block/rbd.c
@@ -1125,7 +1125,7 @@ static int qemu_rbd_load_libs(void)
         return -1;
     }
 
-    librbd_handle = g_module_open("/usr/lib64/qemu/librbd.so.1", 0);
+    librbd_handle = g_module_open("librbd.so.1", 0);
     if (!librbd_handle) {
         error_report("error loading librbd: %s", g_module_error());
         return -1;
-- 
1.8.3.1

